src/Calendar.js

```diff
@@ -696,10 +719,19 @@ class Calendar extends React.Component {
       className,
       elementProps,
       date: current,
+      nowAccessor,
+      min,
+      max,
+      scrollToTime,
       length,
       ...props
     } = this.props
 
+    current = current || get(nowAccessor)
+    min = min || dates.startOf(current, 'day')
+    max = max || dates.endOf(current, 'day')
+    scrollToTime = scrollToTime || dates.endOf(current, 'day')
```

Comment:
this all seems like logic change? what is the rational for this, I think it makes more sense to leave these empty.

I'm also not sure defaulting to scrolling to the end of a day is a good choice...i wouldn't expect that behavior

Reply:
Rationale: the current behavior has `min`, `max` and `scrollToTime` set as `defaultProps` to `new Date()`. I think they all do need to be based on the `getNow()` time. My change of `scrollToTime` is just plain **wrong** - it should have been `startOf(current, 'day')`.

I was just mirroring (well, attempting to :-)) the current behavior using the new accessor instead of the hardwired `new Date()` in the default props.

Having said all of that, I can see an argument for leaving all of this alone - let me give it some more thought so I don't waste anyone's time.